Newsletters/Yearn Finance Newsletter #44/Yearn Finance Newsletter #44.md

```diff
@@ -1,95 +1,95 @@
+Lee la propuesta completa en el foro de Yearn [Aquí](https://gov.yearn.finance/t/yip-63-fund-builder-first-legal-activism-dao/11280).
 
-Check Yearn’s transaction to the DAO [here](https://etherscan.io/tx/0x0ec0fc55d6dc51b426a254bf2d6de138b1b9a1c3031f4ab3a7b39439fa004392)
+Echa un vistazo a la transacción realizada por Yearn para el DAO [Aquí](https://etherscan.io/tx/0x0ec0fc55d6dc51b426a254bf2d6de138b1b9a1c3031f4ab3a7b39439fa004392)
 
-## **Product User Survey**
+## **Encuesta a los usuarios**
 
-We have released a new user survey to gauge what you, the community at Yearn, would like to see added.  
-  
-Yearn is always iterating amongst ourselves and with the help of the community, but with this survey, we’d like to take a more hands on approach to seeing what you guys, the community, has in mind to improve Yearn. We have seen exponential growth in the short time we’ve been active, and we hope to continue innovating in the products we put out and add more utility to the YFI token. We think that we can improve the user experience at Yearn tenfold with the help of your thoughts, and thus become the most competitive in the yield aggregation space among the various other sectors we engage in.
+Hemos realizado una nueva encuesta de usuarios para determinar lo que vosotros, los usuarios de la comunidad, quereis ver en Yearn.
+ 
+Yearn se encuentra en constante proceso de iteración con la ayuda de la comunidad, pero con esta encuesta, queremos ponernos manos a la obra para así determinar lo que vosotros, la comunidad, tenéis en mente para mejorar Yearn. Hemos visto crecimiento exponencial en el corto tiempo que llevamos en actividad y esperamos continuar innovando en los productos que lanzamos al mercado y dando mayor utilidad al token YFI. Creemos que podemos mejorar la experiencia de usuario en Yearn con la ayuda de vuestras ideas y de esa forma ser mas competitivos en el mercado de generación de rendimiento así como en los distintos sectores donde nos desarrollamos.
 
-Fill out the survey [here](https://yearnfinance.typeform.com/to/ojp3J8gn).
+Completa la encuesta [Aquí](https://yearnfinance.typeform.com/to/ojp3J8gn).
 
-## **Introducing Four New yVaults**
+## **Hemos introducido 4 nuevas vaults**
 
 ![](image3.jfif)
 
-Four new yVaults are now live on our UI. Deposit today and earn yield on your AAVE, COMP, TUSD, and SUSHI  
-  
-The AAVE, COMP, and SUSHI vaults are earning yield from single-sided staking pools on [universe.xyz](https://universe.xyz/polymorphs), with no exposure to impermanent loss. TUSD is earning yield from Kashi on SushiSwap.
+4 nuevas yVaults están ahora disponibles en nuestra web. Deposita hoy y genera rendimiento con tu AAVE, COMP, TUSD y SUSHI.
+ 
+Las vaults de AAVE, COMP y SUSHI generan rendimiento haciendo staking en [universe.xyz](https://universe.xyz/polymorphs), por lo que no se encuentran expuestos a perdidas impermanentes. TUSD genera rendimiento en Kashi dentro de SushiSwap.
```

Comment:
AS-IS: Las vaults de AAVE, COMP y SUSHI generan rendimiento haciendo staking en [universe.xyz](https://universe.xyz/polymorphs), por lo que no se encuentran expuestos a perdidas impermanentes. TUSD genera rendimiento en Kashi dentro de SushiSwap.


TO-BE: Las vaults de AAVE, COMP y SUSHI generan rendimiento haciendo staking en [universe.xyz](https://universe.xyz/polymorphs), por lo que no se encuentran expuestos a pérdidas impermanentes. TUSD genera rendimiento en Kashi dentro de SushiSwap.